tests/operations.go

```diff
@@ -245,7 +245,7 @@ func (ops *Operations) GetCstorVolumeCountEventually(namespace, lselector string
 		cvCount := ops.GetCVCount(namespace, lselector)
 		return cvCount
 	},
-		60, 10).Should(Equal(expectedCVCount))
+		120, 10).Should(Equal(expectedCVCount))
```

Comment:
I am not sure what is 120 and what is 10. Hence unable to review.

Reply:
120 is the total waiting time in seconds. 10 is the time interval after which the function retries in seconds.